makefiles/tools/openocd-adapters/iotlab.inc.mk

```diff
@@ -0,0 +1,8 @@
+# iotlab-usb debug adapter
+OPENOCD_ADAPTER_INIT ?= -c 'source [find interface/ftdi/iotlab-usb.cfg]'
```

Comment:
This configuration file is already included by the openocd.cfg file of the iotlab-m3 board. But maybe it removed from the openocd.cfg file ?

Reply:
They are used for different things, and `OPENOCD_ADAPTER_INIT` is evaluated before loading the configuration file if you look at `openocd.sh`.